pkg/server/api/logger/v1/service.go

```diff
@@ -0,0 +1,78 @@
+package logger
+
+import (
+	"context"
+	"fmt"
+
+	loggerv1 "github.com/spiffe/spire-api-sdk/proto/spire/api/server/logger/v1"
+	apitype "github.com/spiffe/spire-api-sdk/proto/spire/api/types"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+type Logger interface {
+	logrus.FieldLogger
+	GetLevel() logrus.Level
+	SetLevel(level logrus.Level)
+}
+
+type Config struct {
+	Log         Logger
+	LaunchLevel logrus.Level
+}
+
+type Service struct {
+	loggerv1.UnsafeLoggerServer
+
+	log         Logger
+	launchLevel logrus.Level
+}
+
+func New(config Config) *Service {
+	config.Log.WithFields(logrus.Fields{
+		"LaunchLevel": config.LaunchLevel,
+	}).Info("Logger service configured")
+	return &Service{
+		log:         config.Log,
+		launchLevel: config.LaunchLevel,
+	}
+}
+
+func RegisterService(s grpc.ServiceRegistrar, service *Service) {
+	loggerv1.RegisterLoggerServer(s, service)
+}
+
+func (service *Service) GetLogger(_ context.Context, _ *loggerv1.GetLoggerRequest) (*apitype.Logger, error) {
+	service.log.Info("GetLogger Called")
```

Comment:
tthis logger must not be used to log things... because it is the base one, you need to get the logger from context
```
	ctx := stream.Context()
	log := rpccontext.Logger(ctx)
```
that logger has metadata like service name, and caller information